src/services/users/fetch-users.test.js

```diff
@@ -0,0 +1,21 @@
+const { expect } = require('chai')
+const {fixtures, setupDB } = require('../../../test/utils')
+const db = require('../../db')
+const fetchUsers = require('./fetch-users')
+
+describe('Fetch all users', () => {
+	beforeEach(async () => {
+		await setupDB()
+		await fixtures.insertUser()
+		await fixtures.insertUser()
+	 })
+
+	describe('Success', () => {
```

Comment:
there's not much happening here so we might not need an inner `Success` block yet

Reply:
sure